### PR TITLE
initdata: update doc for digest usage

### PR DESCRIPTION
**Pull request by huoqifeng**
Add digest usage example in ibmse

### CODE REVIEW

File: src/cloud-api-adaptor/docs/initdata.md [hunk 1]
```diff
@@ -166,5 +166,16 @@ It also calculates the digest `/run/peerpod/initdata.digest` based on the `algor
```
Comment (bpradipt) [inline src/cloud-api-adaptor/docs/initdata.md:166]: nit: s/usae/usage

File: src/cloud-api-adaptor/docs/initdata.md [hunk 2]
```diff
@@ -166,5 +166,16 @@ It also calculates the digest `/run/peerpod/initdata.digest` based on the `algor
 
 The digest can be calculated manually and set to attestation service policy before hand if needed. To calculate the digest, use a tool (for example some online sha tools) to calculate the hash value based on the initdata raw string. The calculated sha384 is: `52af3178dd7ad4bf551e629b84b45bfd1fbe1434b980120267181ae3575ea20ca9013b8eadf31d27eed7ff2552d500ef` for above sample.
 
+For example, for ibmse TEE driver, the `se.user_data` can be set as:
```
Comment (bpradipt) [inline src/cloud-api-adaptor/docs/initdata.md:169]: you may want to add a link here to the attestation policy ref in trustee.

File: src/cloud-api-adaptor/docs/initdata.md (hunk 1 above)
Comment (huoqifeng) [reply src/cloud-api-adaptor/docs/initdata.md:166]: Thanks @bpradipt fixed.

File: src/cloud-api-adaptor/docs/initdata.md (hunk 2 above)
Comment (huoqifeng) [reply src/cloud-api-adaptor/docs/initdata.md]: Thanks @bpradipt , added the claims link in trustee repo here.

File: src/cloud-api-adaptor/docs/initdata.md [hunk 3]
```diff
@@ -166,5 +166,16 @@ It also calculates the digest `/run/peerpod/initdata.digest` based on the `algor
...
+For example, for [IBM SE](https://github.com/confidential-containers/trustee/blob/main/attestation-service/docs/parsed_claims.md#ibm-secure-execution-se), the `se.user_data` can be set as:
+```
+allow if {
+    ...
+    input["se.user_data"] == "353261663331373864643761643462663535316536323962383462343562666431666265313433346239383031323032363731383161653335373565613230636139303133623865616466333164323765656437666632353532643530306566"
+    ...
+}
+```
+
+`se.user_data` is the HEX of the initdata digest string `52af3178dd7ad4bf551e629b84b45bfd1fbe1434b980120267181ae3575ea20ca9013b8eadf31d27eed7ff2552d500ef`.
```
Comment (liudalibj) [inline src/cloud-api-adaptor/docs/initdata.md:178]: can we give an example script on how to to convert `52af3178dd7ad4bf551e629b84b45bfd1fbe1434b980120267181ae3575ea20ca9013b8eadf31d27eed7ff2552d500ef` to "353261663331373864643761643462663535316536323962383462343562666431666265313433346239383031323032363731383161653335373565613230636139303133623865616466333164323765656437666632353532643530306566"

Comment (mkulke) [reply src/cloud-api-adaptor/docs/initdata.md:178]: yeah, that's also not obvious to me. I thought initdata.digest hex was already a text representation? if not should we maybe write initdata.digest as raw bytes, so we avoid the double `to_hex(to_hex(initdata_digest))` wrapping?

Comment (huoqifeng) [reply src/cloud-api-adaptor/docs/initdata.md:178]: Yeah, it's more reasonable to use the hash directly, I created a PR https://github.com/confidential-containers/trustee/pull/462 in trustee to revise it.

**Review by mkulke — APPROVED** [review]

**Review by liudalibj — APPROVED** [review]
LGTM

**Pull request merged**